src/buttons/Chip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native';
+import { withTheme } from '../config';
+import Button, { ButtonProps } from './Button';
+
+export type ChipProps = Omit<
+  ButtonProps,
+  'loading' | 'loadingStyle' | 'loadingProps' | 'buttonStyle'
+> & {
+  type?: 'solid' | 'outline';
+};
+
+const Chip: React.FunctionComponent<ChipProps> = (props: ChipProps) => {
+  return (
+    <Button
+      {...props}
+      titleStyle={{ fontSize: 14, paddingHorizontal: 2 }}
```

Comment:
The way these props are done are not quite correct. 2 items:
1. `{...props}` should always come at the bottom so that people can overwrite the defaults.
2. `titleStyle` and other props should extend the `titleStyle` passed in as a prop so that they can extend the style and not just completely overrwrite it.

Reply:
Thanks a lot for that! I've updated the props accordingly.